### PR TITLE
OHRM-90: updated change log

### PR DESCRIPTION
**Pull request by MadhukaHarith92**

### CODE REVIEW

File: CHANGELOG.TXT [hunk 1]
```diff
@@ -13,6 +13,66 @@ specified here
...
+    Unable to install the build for php 7.2 with any mysql version.
+
+New & Changed Features for OrangeHRM ver 4.1
+    New Features
+    ------------
+    Integration with toggl. Users can sync the time entries tracked in toggl to orangehrm time module.
+    Enforcement of having a strong password for the user account.
+    Compatibility with ldap add-on
+    Environment compatibility with php 7.1 & mariadb 10.2
+
```
Comment (rimaz523) [inline CHANGELOG.TXT]: Remove line space here

File: CHANGELOG.TXT [hunk 2]
```diff
@@ -13,6 +13,66 @@ specified here
...
+    Incorrect version number in config file git issue #262
+    Cannot install in mysql 5.7.23, supporting version is equal or lesser version than 5.7.22
+    Web installer: "Help" link should be changed with a new guide
+    When change password by accessing change password link, system getting inaccessible
+
+New & Changed Features for OrangeHRM ver 4.1.1
+    New Features
+    ------------
+    Verification of supported environment during the installation.
+        -supported php version range
```
Comment (rimaz523) [inline CHANGELOG.TXT]: S in "supported" need to be capitalized

File: CHANGELOG.TXT [hunk 3]
```diff
@@ -13,6 +13,66 @@ specified here
...
+        -Chinese (Simplified Han)
+    Support of cli installer
+
+    Bug Fixes
+    ------------
+    "Csrf token validation failed" message appears in login screen even after trying to login with valid credentials git issue #247
+    Incorrect version number in config file git issue #262
+    Cannot install in mysql 5.7.23, supporting version is equal or lesser version than 5.7.22
+    Web installer: "Help" link should be changed with a new guide
+    When change password by accessing change password link, system getting inaccessible
```
Comment (rimaz523) [inline CHANGELOG.TXT]: System becomes inaccessible after changing password by accessing the change password link

File: CHANGELOG.TXT [hunk 4]
```diff
@@ -13,6 +13,66 @@ specified here
...
+        Supported language packs
+        -English
+        -French
+        -German
+        -Spanish (Costa Rica)
+        -Spanish
+        -Dutch
+        -Chinese (Traditional Han)
+        -Chinese (Simplified Han)
+    Support of cli installer
```
Comment (rimaz523) [inline CHANGELOG.TXT]: is this not properly indented?

File: CHANGELOG.TXT (hunk 1 above)
Comment (MadhukaHarith92) [reply CHANGELOG.TXT]: Done.

File: CHANGELOG.TXT (hunk 2 above)
Comment (MadhukaHarith92) [reply CHANGELOG.TXT]: Done

File: CHANGELOG.TXT (hunk 3 above)
Comment (MadhukaHarith92) [reply CHANGELOG.TXT]: Done.

File: CHANGELOG.TXT (hunk 4 above)
Comment (MadhukaHarith92) [reply CHANGELOG.TXT]: Line 19 was incorrectly indented.

**Pull request merged**